pkg/querier/frontend/queryrange/queryrange.pb.go

```diff
@@ -0,0 +1,2232 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
```

Comment:
We haven't traditionally checked in the .pb.go files.
I realise this particular one may be important to your objectives, but if we do one we should do all.

Reply:
Agree, let's discuss. 

Did not try to see how painful it will be for other projects to use it without it being not commited, so need to check that first.

Reply:
Any thoughts?

Reply:
Will try to cover this at the community meeting today.

Reply:
Thanks!

Reply:
My bad - community meeting is tomorrow (2nd May)

Reply:
From the community meeting we decided that we would generally like to check in the code with a couple caveats: https://github.com/cortexproject/cortex/issues/1371

I would say this PR should either not check in the `pb.go` or wait until after the tasks in that issue are completed. I don't want to be in a state where some files are checked in and others are not.